.github/workflows/pytorch112_aarch64.yml

```diff
@@ -0,0 +1,73 @@
+name: pytorch1_12-aarch64
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: '0 17 * * *' # 1:00 am UTC+8:00
+  workflow_dispatch:
+
+jobs:
+  build:
+    if: github.repository == 'alibaba/BladeDISC'
+    # The type of runner that the job will run on
+    runs-on: [self-hosted, aarch64]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.4.0
+    - name: Build Dev Docker
+      run: |
+        set -e
+        git submodule sync
+        git submodule update --depth=1 --init --recursive
+        cp /etc/apt/sources.list .
+        docker build -t disc-dev-cpu-aarch64 \
+          --build-arg BASEIMAGE=ubuntu:20.04 \
+          --build-arg DISC_HOST_TF_VERSION="tensorflow-aarch64==2.8" \
```

Comment:
why do we need TF in pytorch image?

Reply:
`pytorch112_aarch64.yaml` is mainly copy from `pytorch110_aarch64.yaml` except for the version number.
I will check if tf is necessary

Reply:
No need for tensorflow version